browser/src/Services/Browser/BrowserView.tsx

```diff
@@ -182,9 +182,20 @@ export class BrowserView extends React.PureComponent<IBrowserViewProps, IBrowser
         )
     }
 
+    public prefixUrl = (url: string) => {
+        // Regex Explainer - match at the beginning of the string ^
+        // brackets to match the selection not partial match like ://
+        // match http or https, then match ://
+        const hasValidProtocol = /^(https?:)\/\//i
+        if (url && !hasValidProtocol.test(url)) {
+            return `https://${url}`
+        }
+        return url
+    }
+
     private _navigate(url: string): void {
         if (this._webviewElement) {
-            this._webviewElement.src = url
+            this._webviewElement.src = this.prefixUrl(url)
```

Comment:
Awesome! I noticed though that if I navigate multiple times to different websites, like:
`apple.com` <-- works
`onivim.io` <-- doesn't work

I believe that we may need this logic where we set `webviewElement.src` at `_initializeElement` below, as well. Perhaps we could just refactor that to call `this._navigate(this.props.initialUrl)`, so that it can pick up this logic too?

Reply:
@bryphe seemed to be related to using `https` rather than `http` as a prefix when attempting to access `onivim.io` going to and from `apple.com`, `google.com` etc. all seemed to work fine.

I've tweaked it to use http by default in case there are more cases like that, and call this._navigate in initElement

Also changed the way initElement is called as were getting a warning re. impurity in the render and antipatterns I thought that could be contributing but were still getting it (it doesn't really change functionality  and with async rendering is soon to be no longer true)